src/test/java/uk/ac/ebi/embl/api/validation/plan/ValidationPlanTest.java

```diff
@@ -87,16 +87,39 @@ public void testIsInValidationScope_Right() {
   }
 
   @Test
-  public void testIsInValidationScope_Many() {
-    assertTrue(
-        plan.isInValidationScope(
-            new ValidationScope[] {ValidationScope.EMBL, ValidationScope.INSDC}));
+  public void testIsInValidationScopePutffGroup() {
+    assertTrue(plan.validationScope.isInGroup(ValidationScope.Group.PUTFF));
+    assertFalse(plan.validationScope.isInGroup(ValidationScope.Group.PIPELINE));
   }
 
   @Test
-  public void testIsInValidationScope_ManyWrong() {
-    assertFalse(
-        plan.isInValidationScope(
-            new ValidationScope[] {ValidationScope.EPO, ValidationScope.INSDC}));
+  public void testIsInValidationScopePipelineGroup() {
```

Comment:
Replicate what is in the ValidationScope::testValidationScopeIsInGroup() in ValidationPlanTest. The only difference is that ValidationPlanTest creates the ValidationPlan.